assets/company-admin/components/CompanyDetails.jsx

```diff
@@ -30,14 +29,21 @@ function CompanyDetailsComponent({company, showSeatRequestModal, setSection, com
                             <tr colSpan={2 + numSections} className="subheading">
                                 <td>{section.name}</td>
                             </tr>
-                            {Object.values(seats[company._id]).filter((seat) => seat.section === section._id).map((seat) => (
-                                <CompanyDetailsProductRow
-                                    key={seat._id}
-                                    seat={seat}
-                                    showSeatRequestModal={showSeatRequestModal}
-                                    onNameClicked={() => setSection('users', seat._id)}
-                                />
-                            ))}
+
+                            {
+                                (query === null ?
+                                    Object.values(seats[company._id]).filter((seat) => seat.section === section._id) :
+                                    Object.values(seats[company._id]).filter((seat) => seat.section === section._id && seat.name.toString().includes(query))
+                                )
+                                    .map((seat) => (
```

Comment:
I'm a bit confused with the wording here, shouldn't this be rather `product` instead of `seat`?

Reply:
sounds meaningful and gives context